test/powershell/engine/Basic/NativeCommandBytePiping.Tests.ps1

```diff
@@ -63,4 +63,13 @@ Describe 'Native command byte piping tests' -Tags 'CI' {
             ($pipe)?.Dispose()
         }
     }
+
+    It 'Bytes are retained when redirecting to a file' {
+        testexe -writebytes FF > $TestDrive/content.bin | Should -BeNullOrEmpty
+        Get-Content -LiteralPath $TestDrive/content.bin -AsByteStream | Should -Be 0xFFuy
+    }
+
+    It 'Redirecting to $null should emit no output' {
+        testexe -writebytes FF > $null | Should -BeNullOrEmpty
```

Comment:
@SeeminglyScience These 2 tests don't seem to cover the reported issue, where the downstream command is `Out-Default`. Can you please add another test to cover that scenario?

Reply:
Ah! Great catch, you are 100% right. I'll add that test tomorrow, thanks!

Reply:
Done!